### PR TITLE
Added less-technical language in Docker Quickstart

### PR DESCRIPTION
**Pull request by phazonoverload**

### CODE REVIEW

**Review by rijkvanzanten — APPROVED** [review]

File: docs/self-hosted/quickstart.md [hunk 1]
```diff
@@ -66,11 +70,28 @@ file.
...
+cd Desktop/directus
 docker compose up
 ```
 
+== Windows
+
+Open the Command Line and run the following commands one line at a time:
+
+```
+cd /d %userprofile%\Desktop
```
[the remark this answers is not in the document]
Comment (phazonoverload) [reply docs/self-hosted/quickstart.md:89]: It should

File: docs/self-hosted/quickstart.md [hunk 2]
```diff
@@ -66,11 +71,27 @@ file.
...
+cd Desktop/directus
+docker compose up
+```
+
+== Windows
+
+Open the Command Line and run the following commands one line at a time:
 
 ```
+cd /d %userprofile%\Desktop\directus
```
Comment (paescuj) [inline docs/self-hosted/quickstart.md:90]: This won't work on PowerShell which some users might prefer over CMD... Wondering if there would be a command which works in both terminals 🤔

Comment (br41nslug) [reply docs/self-hosted/quickstart.md:90]: Lets avoid the use of `%userprofile%` alltogether. There isnt really a reason to start "recommending/suggesting" an absolute path based on OS right? For linux/mac we're suggesting a relative path so we probably should for windows too.

Comment (phazonoverload) [reply docs/self-hosted/quickstart.md:90]: Is WSL default? I’m thinking about the less technical individual. If it isn’t default, then I would hope that if installed the reader will know to use the Linux instructions

Comment (br41nslug) [reply docs/self-hosted/quickstart.md:90]: > Maybe we instruct them to open WSL before running the same command? That'll also remove the need to have a mac vs windows thing

I dont think this is a viable option as WSL is not enabled by default in windows (very much a developer only feature at the moment)

Comment (br41nslug) [reply docs/self-hosted/quickstart.md:90]: > The default location when you open the terminal in macOS is the user folder.

The same holds true on Windows.

Comment (phazonoverload) [reply docs/self-hosted/quickstart.md:90]: Oh. lol. Okay, this is unnecessarily convoluted and I can change it then

**Review by paescuj — APPROVED** [review]

**Pull request merged**